docs/content/middlewares/headers.md

```diff
@@ -246,7 +248,7 @@ labels:
 [http.middlewares]
   [http.middlewares.testHeader.headers]
     accessControlAllowMethods= ["GET", "OPTIONS", "PUT"]
-    accessControlAllowOrigin = "origin-list-or-null"
+    accessControlAllowOrigin = ["https://foo.bar.org","https://example.org"]
```

Comment:
It's breaking here, and we need to find a non-breaking solution, do you have any idea?

Reply:
Is the issue we can't change the field type without a CRD version bump?

Can I add another field?
So can I do:

```yaml
accessControlAllowOrigin = "origin-list-or-null"
accessControlAllowOriginList = ["https://foo.bar.org","https://example.org"]
```

?

Reply:
For now, CRD don't have scheme so it's a problem.